tests/modbus/test.yaml

```diff
@@ -0,0 +1,49 @@
+requires:
+  min-version: 6.0.0
+
+args:
+  - -k none
+
+checks:
+
+  - filter:
+      count: 1
+      match:
+        event_type: modbus
+        modbus.id: 10
+
+  - filter:
+      count: 2
+      match:
+        event_type: modbus
+        modbus.request.function_code: RdCoils
+
+  - filter:
+      count: 2
+      match:
+        event_type: alert
```

Comment:
Would it be possible to test also the `modbus` part of the alert ?

Reply:
Sure, we can look into this.